File: Swift-C++-IR-Comparison/test/functions/Swift/FunctionInOutParameter.swift
@main
struct MyMain {

    static func main() {
        var someInt = 3
        var anotherInt = 107
        swapTwoInts(&someInt, &anotherInt)
    }

}

func swapTwoInts(_ a: inout Int, _ b: inout Int) {
    let temporaryA = a
    a = b
    b = temporaryA
}

File: AutoTranspiledTests/Swift-to-Cpp/try-0/FunctionInOutParameter.cpp

#include <iostream>

void swapTwoInts(int* a, int* b) {
    int tempA = *a;
    *a = *b;
    *b = tempA;
}
int main()
{
    int someInt = 3;
    int anotherInt = 107;
    swapTwoInts(&someInt, &anotherInt);

    return 0;
}